app/app/start/page.jsx

```diff
@@ -1,13 +1,22 @@
+"use client"
+
 import Button from "@/components/Button";
 import {style, layout} from "@/app/style";
 import Explore from "../../components/Explore.jsx";
 import {faCircleDollarToSlot} from "@fortawesome/free-solid-svg-icons";
 import {faVolumeUp} from "@fortawesome/free-solid-svg-icons";
 import {faLanguage} from "@fortawesome/free-solid-svg-icons";
 import {faComments} from "@fortawesome/free-regular-svg-icons";
+import { useRouter } from 'next/navigation';
 
 function WelcomePage() {
 
+    const router = useRouter();
+
+    const handleSingUp = () => {
+        router.push("/signIn");
+    }
```

Comment:
Let's replace calling router with component Link. That way you component stays server (which if cool for performance), and also Next is able to prefetch pages if you use Link (which is even better for performance)

Reply:
This comment is also (partially) true for signup page and navbar. Please, replace with Link where is possible